daemon/inertiad/containers/containers.go

```diff
@@ -94,10 +94,13 @@ func StopActiveContainers(docker *docker.Client, out io.Writer) error {
 		if container.Names[0] != "/inertia-daemon" {
 			fmt.Fprintln(out, "Stopping "+container.Names[0]+"...")
 			timeout := 10 * time.Second
-			err := docker.ContainerStop(ctx, container.ID, &timeout)
-			if err != nil {
+			if err := docker.ContainerStop(ctx, container.ID, &timeout); err != nil {
 				return err
 			}
+
+			// Archive container
+			docker.ContainerRename(
+				ctx, container.ID, fmt.Sprintf("%s-%d", container.Names[0], time.Now().Unix()))
```

Comment:
What's the point of renaming these containers? Wouldn't we pile up on archived containers in this case if ran for long period?

Reply:
Ran into some issues with duplicate container names in my testing. The reasoning here is twofold:
- unused containers will be removed by our `Prune()` op, which runs at certain points (pre-build, etc)
- archival might be useful for #293

We definitely need a stronger plan for resource management, but I still have some confusions about how the fuck docker itself manages it. It seems like there is a delay in API call -> thing actually happening, as can be seen in the Travis tests periodically failing. Definitely something that needs more research.